src/main/java/bhs/devilbotz/subsystems/Gripper.java

```diff
@@ -32,16 +32,19 @@ public Gripper() {
   /** This method opens the gripper. */
   public void open() {
     doubleSolenoid.set(Value.kForward);
+    System.out.println("Opening");
   }
 
   /** This method closes the gripper. */
   public void close() {
     doubleSolenoid.set(Value.kReverse);
+    System.out.println("Closing");
   }
 
   /** This method sets the grippers speed to 0. */
   public void stop() {
     doubleSolenoid.set(Value.kOff);
+    System.out.println("Stopping");
```

Comment:
Debug code for gripper

Reply:
I don't understand what `kOff` does for a solenoid.  We should figure that out and see if the stop method is needed.

Reply:
I believe stop turns off both solenoids, so keeps the gripper wherever it is currently. Neither the forward or reverse is actuated.

Reply:
Got it.  I think these prints should be removed.  They may fill up console with lots of msgs, or get lost if other print statements fill up console.  We should figure out how to put equivalent status to Shuffleboard.

Reply:
Done. I removed the prints.

Reply:
yep they were just for quick debugging